examples/subscribe_notify_characteristic.rs

```diff
@@ -0,0 +1,118 @@
+use std::time::Duration;
+
+use btleplug::api::{ValueNotification, CharPropFlags};
+use btleplug::platform::{Adapter, Manager};
+use btleplug::api::{Central, Manager as _, Peripheral};
+use std::io::Cursor;
+use std::error::Error;
+use uuid::Uuid;
+use tokio::time;
+use futures::stream::{StreamExt};
+
+const PERIPHERAL_NAME_MATCH_FILTER: &'static str = "Neuro"; // filter BLE device by partial name
+const NOTIFY_CHARACTERISTIC_MATCH_UUID: &'static str = "6E400002-B534-F393-67A9-E50E24DCCA9E"; // subscribe UUID
```

Comment:
It should be possible to make the constants `Uuid`s rather than strings.

Reply:
That struct + method should I use. I'd like to... my previous code used const func to parse uuid value which I don't see in dev now.

Reply:
You can do `Uuid::from_u128(0x6e400002_b534_f393_67a9_e50e24dccA9e)`.